x-pack/test/security_api_integration/tests/audit/audit_log.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import Path from 'path';
+import Fs from 'fs';
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+class FileWrapper {
+  constructor(private readonly path: string) {}
+  async reset() {
+    // "touch" each file to ensure it exists and is empty before each test
+    await Fs.promises.writeFile(this.path, '');
+  }
+  async read() {
+    const content = await Fs.promises.readFile(this.path, { encoding: 'utf8' });
+    return content.trim().split('\n');
+  }
+  async readJSON() {
+    const content = await this.read();
+    return content.map((l) => JSON.parse(l));
+  }
+  // writing in a file is an async operation. we use this method to make sure logs have been written.
+  async isNotEmpty() {
+    const content = await this.read();
+    const line = content[0];
+    return line.length > 0;
+  }
+}
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const retry = getService('retry');
+
+  describe('Audit Log', function () {
+    const logFilePath = Path.resolve(__dirname, '../../fixtures/audit/audit.log');
+    const logFile = new FileWrapper(logFilePath);
+
+    beforeEach(async () => {
+      await logFile.reset();
+    });
+
```

Comment:
Given the inherent complexity of our authentication process, can we also add tests to assert both a successful and failed `user_login` event?